docs/meetings/client/2023_01_27.md

```diff
@@ -0,0 +1,80 @@
+# Client Meeting Minutes
+
+- **Start Time:** 4:00 PM
+- **End Time:** 4:30
+- **Location:** Zoom
+- **Date:** Jan 27th, 2023
+
+## Attendance
+
+- Client:
+  - Nazim
+- Team A:
+  - Veronica Jack
+  - Edouard Eltherington
+  - Logan Parker
+  - Matthew Kuelker
+
+### Attendance Comments
+
+- Other teams present (unclear if all team members were present)
+
+## Team A
+
+- Matt opened the boy demo for Nazim
+
+- Matt shared the Demo:
+  - There seemed to be un-synched lip positions
+  - We used challenge words and maybe some lag from zoom call
+
+- Logan ask Naz about requesting more poses:
+  - Nazim response: 'Its up to you and your creativity
+- Background should remain transparent
+
+- Matt and Logan brought up the language recognizers
+  - Challenge 1: English rec looks better, but takes longer to process
+
+Would you prefer effciency or quality ?
+
+- Naz says that quality is critical
+
+- Logan drops to clips in the discord chat
+  - One with english rec
+  - One with non-english rec
+
+- In terms of implementation Kuk uses a Docker server
+  - Logan mentions that we have a bottleneck in our local Dock implementaion
+
+## Team B
+
+- Cooper Smith opened a demo
+
+- Demo looked good:
+  - Includes single frame poses and blinking
+  - Poses were set be comma/line
+  - Flow looks great!
+  - Locally the video takes 30-45 seconds
+  - Plan to have cloud computed mechanism to speed up the process
+  - Seems like group is using Gentle, not sure
+  - Naz wanted poses to be change more ofter
+- Overall Nazim enjoyed the demo
+
+## Team C
+
+- Audio was not working
+- Showed demos, both were quick
+  - Implementation: using a series of JSON files
+  - No limit on mouth textures 
+  - Using static textures on overlays
+  - Only took a couple seconds to render video
+  - Asks about poses for the video, 
+
+# Comments
+
+- There are many avatars because Naz wants programs to be very generic

```

Comment:
He wants us to just test using 1 avatar, but wants to ensure that it could be easily extended to other avatars